atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -74,6 +148,11 @@ def run(self):
                                  registry_noschema)
                 continue
 
+            if push_conf_registry.insecure:
+                # override insecure if passed
+                insecure = push_conf_registry.insecure
```

Comment:
Should `push_conf_registry` should be initialised to `None` before the loop? Currently static analysis tools complain that this might not be defined.